tomviz/AddPythonTransformReaction.cxx

```diff
@@ -219,6 +219,35 @@ OperatorPython* AddPythonTransformReaction::addExpression(DataSource* source)
           source->addOperator(op);
       }
   }
+  else if (scriptLabel == "Sobel Filter") //UI for Sobel Filter
+  {
+      QDialog dialog(pqCoreUtilities::mainWidget());
+      QHBoxLayout *layout = new QHBoxLayout;
+      QLabel *label = new QLabel("Axis:");
+      layout->addWidget(label);
+      QSpinBox *axis = new QSpinBox;
+      axis->setRange(0, 2);
+      axis->setValue(0);
```

Comment:
Agreed. I need to go back and do that for my "Rotate" code as well—probably set a bad example.